src/datasets/resource_scripts/cellxgene_census_tower.sh

```diff
@@ -1,20 +1,121 @@
 #!/bin/bash
 
+# template for adding new datasets
+#   - id: cxg_
+#     species:
+#     census_version: "2023-07-25"
+#     obs_value_filter: "dataset_id == ''"
+#     obs_batch:
+#     dataset_name:
+#     dataset_summary:
+#     dataset_description:
+#     dataset_url:
+#     dataset_reference:
+#     dataset_organism:
+
+# not sure which dataset ids to use
+#   - id: cxg_human_brain_atlas
+#     species: homo_sapiens
+#     census_version: "2023-07-25"
+#     obs_value_filter: "dataset_id == ''" # <--- ?
```

Comment:
It might be good to test cases where the user wants all datasets under a collection_id. Is there a smart query for the `get_anndata` function?

Reply:
not with what's inside the obs in the soma object. Afaik these are the fields that can be used for the query:

```python
with cellxgene_census.open_soma() as conn:
  exp = conn["census_data"]["homo_sapiens"]
  # obs = exp.obs.read().concat().to_pandas()

exp.obs.keys()
```

    ('soma_joinid', 'dataset_id', 'assay', 'assay_ontology_term_id', 'cell_type', 'cell_type_ontology_term_id', 'development_stage', 'development_stage_ontology_term_id', 'disease', 'disease_ontology_term_id', 'donor_id', 'is_primary_data', 'self_reported_ethnicity', 'self_reported_ethnicity_ontology_term_id', 'sex', 'sex_ontology_term_id', 'suspension_type', 'tissue', 'tissue_ontology_term_id', 'tissue_general', 'tissue_general_ontology_term_id')